packages/storage-adapter-local-storage/README.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
Also, all the READMEs are getting rewritten soon. Now that there's a docs site, I don't like just generating them from api docs.